extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java

```diff
@@ -51,18 +32,6 @@ public ContainerImageInfoBuildItem(Optional<String> registry, String group, Stri
         return registry;
     }
 
-    public String getGroup() {
-        return group;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getTag() {
-        return tag;
-    }
```

Comment:
I've just found that removing these methods is actually SPI-breaking :-/ It was a bad SPI arguably, but still. Something to bear in mind for the future.

(And, well, I'll dig out the `name` out of the `image` :-) )

Reply:
:+1: 

Better break now that no one is using it :P

Reply:
Thanks for being so considerate :stuck_out_tongue_closed_eyes:

Reply:
:laughing: